src/main/java/br/unb/cic/analysis/df/OverridingAssignmentAnalysis.java

```diff
@@ -70,15 +72,20 @@ protected void flowThrough(FlowSet<DataFlowAbstraction> in, Unit u, FlowSet<Data
     @Override
     protected FlowSet<DataFlowAbstraction> gen(Unit u, FlowSet<DataFlowAbstraction> in) {
         FlowSet<DataFlowAbstraction> res = new ArraySparseSet<>();
+
+        filterJInstanceFields(u);
```

Comment:
This method is generating the hashmaps for JinstanceField and StaticFieldRef, I think it should change the name. I also don't think that filter fits what the method is doing.

* Don't forget to adjust the comments if necessary.

Reply:
Done.